app/workers/audience_segment_refresh_all_worker.rb

```diff
@@ -8,6 +8,8 @@ def perform
       .distinct(:audience_segment_id)
       .pluck(:audience_segment_id).compact
 
-    AudienceSegmentRefreshWorker.perform_bulk([ids])
+    ids.each do |id|
+      AudienceSegmentRefreshWorker.perform_async(id)
+    end
```

Comment:
```suggestion
    AudienceSegmentRefreshWorker.perform_bulk(ids)
```
I'm not sure how large the ids get but this optimization is a good idea.

Reply:
@maestromac this is currently causing the bug linked in the Honeybadger issue (and has been for a while but we've sort of been ignoring it)

The inputs that we're passing aren't compatible with the expected inputs — I'm not _entirely_ sure how to reconcile it and my fix is sort of hacky in this sense.

You're welcome to fix this properly by adjusting the input so it works as expected.

FWIW the array we're currently sending is of length _two_ so it's not a big deal from an optimization perspective, but I agree with you in general.

Wanna edit this PR to make the passed variables work instead of my change?

Reply:
Ok I see. I think the error is caused by nested array.

Reply:
@maestromac Thanks!